### PR TITLE
Minor updates:

### PR DESCRIPTION
**Pull request by saradickinson**
- remove DoH from terminology because it is not used later in the document
- be clearer that the secondary is the IXoT client
- move text on port choose up in section 5.2 to better match 5.3
- add question a condensation of responses
- clarify port requirement for ACL control

### CODE REVIEW

File: draft-ietf-dprive-xfr-over-tls.xml [hunk 1]
```diff
@@ -362,13 +360,19 @@ applicable to XFR-over-TLS as well.
 <t>It is RECOMMENDED that clients and servers that support XoT also implement
 EDNS0 Keepalive [RFC7828].
 </t>
+<t>It is useful to note that in these mechanisms is it the secondary that initiates the TLS connection to the primary for a XFR request, so that in terms of connectivity the secondary is the TLS client and the primary the TLS server.
```
Comment (ShivanKaul) [inline draft-ietf-dprive-xfr-over-tls.xml:363]: "It is useful to note that in these mechanisms is it" -> it is

Comment (saradickinson) [reply draft-ietf-dprive-xfr-over-tls.xml:363]: Fixed - thanks

File: draft-ietf-dprive-xfr-over-tls.md [hunk 2]
```diff
@@ -323,12 +321,18 @@ applicable to XFR-over-TLS as well.
 It is RECOMMENDED that clients and servers that support XoT also implement
 EDNS0 Keepalive [RFC7828].
 
+It is useful to note that in these mechanisms is it the secondary that initiates the TLS connection to the primary for a XFR request, so that in terms of connectivity the secondary is the TLS client and the primary the TLS server.
```
Comment (wtoorop) [inline draft-ietf-dprive-xfr-over-tls.md:324]: s/is it/it is/

Comment (saradickinson) [reply draft-ietf-dprive-xfr-over-tls.md:324]: Ha - I fixed this in the other PR but not this one!

**Pull request merged**